### PR TITLE
Add OpenMP flags to PLASMA+SLATE example

### LINKED ISSUE #38: plasma example build error [with xsdk@0.8.0]
[spack-build-out.txt](https://github.com/xsdk-project/xsdk-examples/files/11355537/spack-build-out.txt)

Ref:
```
balay@xsdk:/data/balay/spack> nice ./bin/spack install  --test=root --fail-fast -j32 xsdk-examples@0.4.0%gcc@9.2.0+cuda cuda_arch=70 ^cuda@11.6.0 ^openmpi ^xsdk@0.8.0+trilinos
```

### PR DESCRIPTION
**Pull request by luszczek**
Fixes #38 by adding flags specific to OpenMP

### CODE REVIEW

**Comment by balay** [comment]
the commit in this PR was over `testing-v0.4.0`

I rebased it onto `master` and then merged to  `testing-v0.4.0`

**Comment by v-dobrev** [comment]
Merged in branch [testing-v0.4.0](https://github.com/xsdk-project/xsdk-examples/tree/testing-v0.4.0) for testing ...

**Comment by v-dobrev** [comment]
Re-merged in branch [testing-v0.4.0](https://github.com/xsdk-project/xsdk-examples/tree/testing-v0.4.0) for testing ...

**Comment by v-dobrev** [comment]
After some discussions in #46, we should consider updating this PR to use `slateConfig.cmake` instead of the `cmake/FindSlate.cmake` module. If we do that, there will be no need to explicitly link the PLASMA example with OpenMP -- that dependency will come from `slateConfig.cmake`.

Any thoughts?

File: plasma/CMakeLists.txt
[the remark this answers is not in the document]
Comment (v-dobrev) [reply plasma/CMakeLists.txt:1]: @mgates3,

If I understand you correctly, you are saying that we can simply remove the lines
```cmake
if(NOT ENABLE_CUDA)
    target_compile_definitions(ex1solve PRIVATE SLATE_NO_CUDA)
endif()
target_compile_definitions(ex1solve PRIVATE SLATE_NO_HIP)
```
from this file, right?

[the remark this answers is not in the document]
Comment (mgates3) [reply plasma/CMakeLists.txt:1]: @v-dobrev Yes.

**Pull request merged**